### PR TITLE
add compat data for PermissionStatus

### PR DESCRIPTION
**Pull request by gsouquet**
https://developer.mozilla.org/docs/Web/API/PermissionStatus

### CODE REVIEW

File: api/PermissionStatus.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
+{
+  "api": {
+    "AbortSignal": {
```
Comment (connorshea) [inline api/PermissionStatus.json]: This isn't the right API name :)

**Comment by connorshea** [comment]
This doesn't include [status](https://developer.mozilla.org/en-US/docs/Web/API/PermissionStatus/status), though that might be intentional since it's deprecated? I'd still say it's useful since some browsers still seem to implement it.

**Comment by gsouquet** [comment]
Thank @connorshea for the review, fixed the API name.
Regarding `status` it is referenced as an `alternative_name` in `state`. The spec has evolved and the Chrome implementation was initially with `status` and moved away from it.
Hope that makes sense

**Comment by ddbeck** [comment]
This looks good. Thanks for the PR, @gsouquet, and reviewing it, @connorshea! 🎉

**Pull request merged**